packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -89,12 +93,23 @@ const BpkGraphicPromo = (props: Props) => {
   } = props;
 
   // FIXME: Use useCallback() here when React is updated.
-  const onClickWrapper = (event: React.MouseEvent<HTMLElement>) => {
-    event.stopPropagation();
+  const onClickWrapper = (
```

Comment:
We can start to useCallback() here now so not sure if that will impact the changes here?

Reply:
No impact, but happy to do this.

Reply:
I tried this and hooks don't work with unit tests for some reason. I will revert this.